tests/integration/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2019 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+cmake_minimum_required(VERSION 3.5)
```

Comment:
probably this should be enough in root CMakeLists.txt

Reply:
every other cmake list explicitly states required minimum version of cmake, why this should be different?

Reply:
Might be a copy paste error.
If you only go trough root CMakeLists.txt then you dont need it. And currently we only go trough root CMakeLists.txt

Reply:
I'd say consistency is more important that avoiding style that does no harm.